src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -180,4 +180,10 @@
   <data name="SARIF1014_UriBaseIdRequiresRelativeUri" xml:space="preserve">
     <value>If a fileLocation object contains a "uriBaseId" property, the value of the "uri" property must be a relative URI reference.</value>
   </data>
+  <data name="SARIF1015_Default" xml:space="preserve">
```

Comment:
>SARIF1015_Default [](start = 14, length = 17)

It would be nice if we had a convention where every message was followed by a reference to the relevant section of the specification. This would be difficult to maintain but would be extremely helpful. We could consider making a pass towards the end when the spec ships. Alternately, we could contribute a bookmarking scheme that allows us to publish a stable identifier to a web-hosted spec.

Reply:
That's a really nice idea. I agree it's too early, because section numbers keep changing, but yes, if we had a stable URI that would help.

---
In reply to: [211694972](https://github.com/Microsoft/sarif-sdk/pull/1040#discussion_r211694972) [](ancestors = 211694972)